riscv/insns/ssrdp.h

```diff
@@ -0,0 +1,7 @@
+#include "zicfiss.h"
+require_extension(EXT_ZICFISS);
+
+if (!xSSE()) {
+  #include "mop_r_N.h"
```

Comment:
Since for all other instructions we code as "if (xSSE)", maybe we code ssrdp also that way to be consitent?

Reply:
Thanks. The reasoning is consistent.